docs/ref/forms/api.txt

```diff
@@ -1210,24 +1210,33 @@ Methods of ``BoundField``
 
 .. method:: BoundField.label_tag(contents=None, attrs=None, label_suffix=None)
 
+    Renders the label tag of a form field with the template specified by
+    :attr:`.Form.template_name_label`.
+
+    The available context is:
+
+    * ``field``: The bound field
+    * ``contents``: By default a concatenated string of
+      :attr:`BoundField.label` and :attr:`Form.label_suffix`. The can be
+      overridden by the ``contents`` and ``label_suffix`` arguments.
+    * ``attrs``: A ``dict`` containing ``for``, :attr:`Form.required_css_class`
+      and ``id``. ``id`` is generated by the field's widget ``attrs`` or
+      :attr:`Form.auto_id`. Additional attributes can be provided by the
+      ``attrs`` argument.
+    * ``use_tag``: A bool which is ``True`` if the label has an ``id``.
```

Comment:
`bool` => `boolean` I think? A brief, non-exhaustive, check of the docs folder indicates that's the prevalent form.